test/message_extraction/part_of_sample_with_messages.dart

```diff
@@ -13,20 +13,19 @@ class Person {
 class YouveGotMessages {
 
   // A static message, rather than a standalone function.
-  static staticMessage() => Intl.message("This comes from a static method",
-      name: 'staticMessage');
+  static staticMessage() =>
+      Intl.message("This comes from a static method", name: 'staticMessage');
 
   // An instance method, rather than a standalone function.
-  method() => Intl.message("This comes from a method", name: 'method',
-      desc: 'This is a method with a '
-            'long description which spans '
-            'multiple lines.');
+  method() => Intl.message("This comes from a method",
+      name: 'method', desc: 'This is a method with a '
+      'long description which spans '
+      'multiple lines.');
 
   // A non-lambda, i.e. not using => syntax, and with an additional statement
   // before the Intl.message call.
   nonLambda() {
     // TODO(alanknight): I'm really not sure that this shouldn't be disallowed.
-    var x = 'something';
```

Comment:
I'm not sure how to write the test in such a way that it fails if this is removed.